modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -0,0 +1,116 @@
+--
+
+<2> This optional attribute specifies the icon for the menu item. You can use default icons or extend the icon set with dynamic plugins. {product-short} also provides additional icons in its internal library, such as: 
++
+--
+.Icons in the internal library =
+[source]
+----
+  home: from '@mui/icons-material/HomeOutlined'
+  group: from '@mui/icons-material/People'
+  category: from '@mui/icons-material/CategoryOutlined'
+  extension: from '@mui/icons-material/ExtensionOutlined'
+  school: from '@mui/icons-material/SchoolOutlined'
+----
+
+If the icon is already defined in the `dynamicRoutes` configuration under `menuItem.icon`, it can be removed from the in the `menuItems` configuration. Also, both SVG and HTML image icons are supported. For example:
+
+.Example SVG icon
+[source,html]
+----
+icon: <svg width="20px" height="20px" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" fill="#ffffff">...</svg>
+----
+
+.Example image icon
+[source,html]
+----
+icon: https://img.icons8.com/ios-glyphs/20/FFFFFF/shop.png
+----
+--
+
+<3> This optional attribute specifies the title of the menu item. It can be removed if the title is already specified in the `dynamicRoutes` configuration under `menuItem.text`.
+
+<4> This optional attribute sets the order in which menu items appear in the sidebar. The default priority is 0, which places the item at the bottom of the list. A higher priority value places the item higher in the sidebar.
+
+<5> This optional attribute specifies the parent menu item under which the current item is nested. If this attribute is used, the parent menu item must be defined elsewhere in the `menuItems` configuration of any enabled plugin.
+
+The following is an example of `menuItems` configuration in `app-config-rhdh.yaml` file:
+
+.Example `menuItems` configuration
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    <package_name>:
+      dynamicRoutes:
+        - path: /my-plugin
+          module: CustomModule
+          importName: FooPluginPage
+          menuItem:
+            icon: fooIcon
+            text: Foo Plugin Page
+      menuItems:
+        my-plugin: # matches `path` in `dynamicRoutes`
+          priority: 10 # controls order of plugins under the parent menu item
+          parent: favorites # nests this plugin under the `favorites` parent menu item
+        favorites: # configuration for the parent menu item
+          icon: favorite # icon from RHDH system icons
+          title: Favorites # title for the parent menu item
+          priority: 100 # controls the order of this top-level menu item
+----
+
+Also, to ensure that a menu item is identified as a main menu item, you must add `default.` prefix to its key. For example:
```

Comment:
Consider transforming it in a step.

Reply:
Just a suggestion not a request, but when I asked to start this 7.3 section with two parts, I thought we could explain common things together and explain each part in order. Now this main menu items part is far from where the first time it's been mentioned. And if user is looking for this piece of information they can't find it unless finish reading the whole 7.3 section.

Reply:
Done, this section has been converted into steps.